Automattic/wp-calypso
# Jetpack Connect: Separate LoggedInForm from AuthorizeForm

Separate `LoggedInForm` from `authorize-form`.

To test:

* Verify that there are no regressions in the authorize step (`/jetpack/connect/authorize`)
  * logged in
  * logged out

Simple connection test flow instructions: P7rd6c-Me-p2

- [X] Pending completion after #13752 is merged.

Part of: Separate jetpack-connect from signup #12991
